src/index.js

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import firebase from 'firebase/app';
+import { Provider } from 'react-redux';
 import 'firebase/auth';
 
 import './index.css';
 
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import firebaseCred from './firebase-cred.json';
+import configureStore from './store/configureStore';
```

Comment:
Missing file

Reply:
oh my bad. forgot the new files :|
pushed now

Reply:
and i think the builds are also passing now

Reply:
great, thanks @dnivra26 - will you be sending more PRs regarding redux?

Reply:
this PR is just adding redux to the project. If you can tell how we are planning to use redux, like what to store in the redux store, should we use middlewares like thunk, etc.. i can help

Reply:
@dnivra26 I am not sure tbh, I don't have experience working with redux, so I am keen to know your opinion, ie what you think is best.

Reply:
cool. no problem. ill go through the app and the code in detail tomorrow and come up with my recommendations

Reply:
Awesome, will you push the changes to the same branch?

Reply:
probably as part of different PRs

Reply:
sure